debian/changelog

```diff
@@ -1,3 +1,9 @@
+containerd.io (1.7.21-1) release; urgency=medium
+
+  * Update containerd binary to v1.7.21
```

Comment:
This also switches to go runtime 1.22.6; I think we included such updates in the changelog as well;

```
* Update Golang runtime to 1.22.6.
 ```